src/components/URLInput/URLInput.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import Grid from "@material-ui/core/Grid";
+import Textfield from "@material-ui/core/Textfield";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core";
+import "./URLinput.css";
+/* resolve CSS */
+/*error toggle needs implementing*/
+/*Button onClick needs to hide input and open results*/
+
+const useStyles = makeStyles(() => ({
+  inputURLArea: {
+    paddingTop: "25px",
+    paddingBottom: "55px",
+  },
+
+  inputBox: {
+    textAlign: "center",
+    marginBottom: "30px",
+    fontFamily: "Graduate, sans-serif",
+  },
+
+  inputButton: {
+    background: "#6bb26d",
+    borderRadius: 0,
+    fontFamily: "Graduate,",
+    fontSize: "20px",
+  },
+}));
+
+function URLInput(props) {
+  const [text, setText] = useState("");
+  const classes = useStyles();
+
+  const handleTextChange = (event) => {
+    setText(event.target.value);
+  };
+
+  const handleClick = () => {
+    props.getURL(text);
```

Comment:
@abirdonwire Should the `getURL` function exist within the URLInput component as opposed to in App? Or does it need to exist at all? It looks like we 'set' the text state, but then don't use it when it comes to handling a click (instead we use the getURL function - but could just use the 'text' variable)?

Reply:
Want to have a look with me tomorrow? I did that function initially before we had the API connected so if it's defunct now we have setState then that's fine.